creator-node/src/config.js

```diff
@@ -126,14 +126,9 @@ const config = convict({
     default: null
   },
 
-  maxExportClockValueRange: {
-    doc: 'Maximum range of clock values to export at once to prevent process OOM',
-    format: Number,
-    env: 'maxExportClockValueRange',
-    default: 10000
-  },
-
-  // Rate limit configs
+  /**
```

Comment:
all changes are just re-ordering + grouping, except for `minimumSecondaryUserSyncSuccessPercent` on line 559